spec.emu

```diff
@@ -678,7 +805,11 @@ contributors: Ujjwal Sharma, Younies Mahmoud
                 1. If _v_ is not *undefined*, set _v_ to 𝔽(_v_).
               1. Else,
                 1. Assert: _v_ is not *undefined*.
-              1. Perform ! CreateDataPropertyOrThrow(_options_, _p_, _v_).
+              1. If _v_ is *"fractional"*, then
```

Comment:
Instead of checking v everytime and asserting the slot value for the current row (especially since it's not entirely necessary, you can instead check p), why not make the whole thing conditional on p?

Reply:
Not sure I get this one. If I make it conditional on `_p_` then we'll be treating non-fractional styles as if they're fractional.